src/lib/containers/access_requirement_list/AccessRequirementList.stories.tsx

```diff
@@ -0,0 +1,270 @@
+            concreteType:
+              'org.sagebionetworks.repo.model.SelfSignAccessRequirement',
+            isApproved: false,
+          },
+        },
+      ]),
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${ACCESS_APPROVAL}`,
+        async (req, res, ctx) => {
+          const response: AccessApproval = {
+            submitterId: MOCK_USER_ID.toString(),
+            accessorId: MOCK_USER_ID.toString(),
+            state: ApprovalState.APPROVED,
+          }
+          return res(ctx.status(201), ctx.json(response))
+        },
+      ),
+
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${RESEARCH_PROJECT}`,
+        async (req, res, ctx) => {
+          return res(ctx.status(201), ctx.json({}))
+        },
+      ),
+      rest.get(
+        `${MOCK_REPO_ORIGIN}${ACCESS_REQUIREMENT_RESEARCH_PROJECT_FOR_UPDATE(
+          ':id',
+        )}`,
+        async (req, res, ctx) => {
+          const response: ResearchProject = {
+            id: '1981321',
+            accessRequirementId: req.params.id.toString(),
+            institution: 'Black Mesa Research Facility',
+            projectLead: 'Gordon Freeman',
+            intendedDataUseStatement:
+              'We plan to use this data to investigate the properties of antimatter. Our findings will be published in' +
+              ' a peer-reviewed journal, and derivative data will be shared on Synapse.',
+          }
+          return res(ctx.status(200), ctx.json(response))
+        },
+      ),
+      rest.get(
+        `${MOCK_REPO_ORIGIN}${ACCESS_REQUIREMENT_DATA_ACCESS_REQUEST_FOR_UPDATE(
+          ':id',
+        )}`,
+        async (req, res, ctx) => {
+          const response: RequestInterface = {
+            accessRequirementId: req.params.id,
+            concreteType: 'org.sagebionetworks.repo.model.dataaccess.Request',
+          }
+          return res(ctx.status(200), ctx.json(response))
+        },
+      ),
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${DATA_ACCESS_REQUEST}`,
+        async (req, res, ctx) => {
+          return res(
+            ctx.status(201),
+            ctx.json({
+              id: '61561981',
+              etag: '0000',
+            }),
+          )
+        },
+      ),
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${DATA_ACCESS_REQUEST_SUBMISSION(':id')}`,
+        async (req, res, ctx) => {
+          return res(ctx.status(201), ctx.json({}))
+        },
+      ),
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${FILE_HANDLE_BATCH}`,
+        async (req, res, ctx) => {
+          const response: BatchFileResult = {
+            requestedFiles: [
+              {
+                fileHandleId: mockFileHandle.id,
+                fileHandle: mockFileHandle,
+              },
+            ],
+          }
+          return res(ctx.status(201), ctx.json(response))
+        },
+      ),
+    ],
+  },
+}
+
+HasUnmetRequirements.args = {
+  entityId: mockFileEntity.id,
+  renderAsModal: true,
+}
```

Comment:
Still have other stories we could write, like other types of Access Requirements or the renewal flow